cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -173,6 +173,14 @@ spec:
               properties:
                 enable:
                   type: boolean
+            cleanupPolicy:
+              properties:
+                deleteDataDirOnHosts:
+                  type: string
+                  pattern: ^$|^destroy-data-when-the-cluster-is-deleted$
```

Comment:
How about `yes-really-destroy-data`? It's growing on me more than what I suggested before. The fact that it happens on cluster deletion seems clear enough since we're in the cleanup policy.